server/api/posts.js

```diff
@@ -0,0 +1,28 @@
+import { Router } from 'express';
```

Comment:
Lines 15-21 are what actually allow for clicking on a post on the ~/posts/ page and redirect you to localhost:5000/post/this-is-the-slug (see /:slug in line 15). We actually want clicking on an ArticleCard to take you to a discussion view. This has not yet been implemented. Because this feature isn't exactly what we want, we don't need to move anything in this file into app.js or router.js.